services/ingest-file/ingestors/settings.py

```diff
@@ -23,3 +23,6 @@
 # List available NER models
 NER_MODELS = set(env.to_list('INGESTORS_NER_MODELS', ['eng']))
 NER_DEFAULT_MODEL = 'xx'
+
+# Use the environment variable set in aleph.env
+BALKHASH_DATABASE_URI = env.get('ALEPH_DATABASE_URI')
```

Comment:
That is how I solved it in my deployment (by setting BALKHASH_DATABASE_URI) in the aleph.env file however I personally found that confusing that I had to set 2 different DB URI's.  I thought it might make more sense to just use the one ALEPH_DATABASE_URI.  

If you disagree though I can remove this and an example in the aleph.env.templ with a comment that both DB URI's have to be set.

Reply:
Sorry, I was unclear :/ It's just that placing this in the `ingestors/settings.py` will have no effect. I'll make a quick amend after merging this and link it back here for context!

Reply:
cf. https://github.com/alephdata/aleph/pull/1098/commits/a1bfba54c4c1c100201c6f077f617eb6da2a1e1f#diff-9817c107df519a448be8a73e33a89f02R29